src/PrestaShopBundle/Resources/config/services/core/product.yml

```diff
@@ -24,3 +25,7 @@ services:
       - '@=service("prestashop.adapter.legacy.configuration").get("PS_ATTRIBUTE_ANCHOR_SEPARATOR")'
 
   PrestaShop\PrestaShop\Core\Product\Combination\NameBuilder\CombinationNameBuilderInterface: '@PrestaShop\PrestaShop\Core\Product\Combination\NameBuilder\CombinationNameBuilder'
+
+  prestashop.core.product.combination.name_builder.combination_name_builder:
```

Comment:
Why re-introduce this old service name if it's not used anymore? Seems like it wasn't used in 8.0 either so no need to add an already deprecated not used service